packages/beacon-node/src/execution/engine/http.ts

```diff
@@ -428,19 +428,19 @@ export class ExecutionEngineHttp implements IExecutionEngine {
 
     switch (newState) {
       case ExecutionEngineState.ONLINE:
-        this.logger.info("ExecutionEngine became online");
+        this.logger.info("Execution client became online");
         break;
       case ExecutionEngineState.OFFLINE:
-        this.logger.error("ExecutionEngine went offline");
+        this.logger.error("Execution client went offline");
         break;
       case ExecutionEngineState.SYNCED:
-        this.logger.info("ExecutionEngine is synced");
+        this.logger.info("Execution client is synced");
         break;
       case ExecutionEngineState.SYNCING:
-        this.logger.info("ExecutionEngine is syncing");
+        this.logger.warn("Execution client is syncing");
```

Comment:
```suggestion
        this.logger.warn("Execution client is syncing hence client would not be able to participate in the network till its synced");
```

Reply:
do you really feel like we need to add this? We are not printing this out when the beacon node itself is syncing, my general feeling about this is that users/operators generally understand what a syncing client implies.

Reply:
> hence client would not be able to fully participate in the network till its synced

@g11tech on this topic, what does "not fully" mean in that case, I saw users complain about missed head votes which was due to EL lacking behind / unsynced.

My understanding is
1. if EL is really far behind --> missed attestations / proposals
2. if EL is close to being synced (~ 1-2 epochs behind) --> incorrect / missing head votes, att target / source votes still work, missed proposals
3. if EL is slow to process blocks (can't quite keep up), same as degraded performance as point 2

Reply:
if el is not synced with the cl head then except gossip we do not participate in the network at all including attests, proposals aggregates etc. so none of validator duties

Reply:
> we do not participate in the network at all

In that case I think it should be fine to just print "Execution client is syncing" as warn. Maybe the scenario where all head votes are missed is related to the CL not being able to keep up with head (but close to it).

Reply:
I'm not a fan really of long explanations in the logs. I think something more concise would be: `Execution client is syncing. Validator duties on hold.`

Reply:
> I'm not a fan really of long explanations in the logs

Yeah, I think we should keep logs simple and use commonly understood terms. For errors that crash the process or fail a CLI command we can add more details as it is a one time message and we should help the user to fix the issue without having to guess what went wrong or having to ask in discord because the error is unclear.

> Validator duties on hold

hmm personally not a fan, would have to add this to offline state change message as well and then add another message to online, synced that indiciates that those are not "on hold" anymore.